examples/lifelong_learning/atcii/README.md

```diff
@@ -0,0 +1,177 @@
+# Using Lifelong Learning Job in Thermal Comfort Prediction Scenario
+
+This document introduces how to use lifelong learning job in thermal comfort prediction scenario. 
+Using the lifelong learning job, our application can automatically retrains, evaluates, 
+and updates models based on the data generated at the edge.
+
+##  Thermal Comfort Prediction Experiment
+
+
+### Install Sedna
+
+Follow the [Sedna installation document](/docs/setup/install.md) to install Sedna.
+
+### Prepare Dataset
+In this example, you can download ASHRAE Global Thermal Comfort Database II to initial lifelong learning Knowledgebase .
```

Comment:
better to give this dataset original source

Reply:
fix